tt_metal/impl/dispatch/hardware_command_queue.cpp

```diff
@@ -31,7 +31,7 @@
 #include "event/dispatch.hpp"
 #include "hal_types.hpp"
 #include "logger.hpp"
-#include "program_device_map.hpp"
+#include "tt_metal/impl/program/program_device_map.hpp"
```

Comment:
```suggestion
#include "program/program_device_map.hpp"
```

See above comment for details.  'cept in this case we're still within `impl`.  So public and private inc dirs are both permitted.  But those dirs that leak outside the scope of `impl`'s dir need to get eradicated.  I'll sweep them someday when I have a bit of time.